### PR TITLE
Make bitmex stream symbol aware

### PR DESCRIPTION
**Pull request by bonomat**
Once we have #2615 we can plug it all together.

### CODE REVIEW

File: daemon/src/lib.rs [hunk 1]
```diff
@@ -417,9 +417,12 @@ where
 
     #[instrument(skip(self), err)]
     pub async fn propose_settlement(&self, order_id: OrderId) -> Result<()> {
+        // TODO: we need to know which contract symbol the order is for
```
Comment (luckysori) [inline daemon/src/lib.rs:420]: Ahh, this is pretty annoying!

I don't want us to have to pass it in via the HTTP API or load it from the database. Shall we move the quote-fetching to the `taker_cfd::Actor` already?

File: daemon/src/projection.rs [hunk 2]
```diff
@@ -1199,8 +1199,11 @@ impl xtra::Actor for Actor {
                 loop {
                     {
                         let span = info_span!("Update projection with latest quote");
+                        // TODO: subscribe to all the quotes
```
Comment (luckysori) [inline daemon/src/projection.rs:1202]: 🙃 Shall we already adapt the `State` to handle more than one `Quote`?

**Review by luckysori — APPROVED** [review]
LGTM. I left a couple of suggestions/questions regarding things that maybe could be done already.

File: daemon/src/projection.rs (hunk 2 above)
Comment (bonomat) [reply daemon/src/projection.rs:1202]: That's done in #2615

**Comment by bonomat** [comment]
bors retry

**Pull request merged**